web/elm/src/Build/Styles.elm

```diff
@@ -141,10 +142,19 @@ triggerTooltip =
     ]
 
 
-stepHeader : List (Html.Attribute msg)
-stepHeader =
+stepHeader : StepState -> List (Html.Attribute msg)
+stepHeader state =
     [ style "display" "flex"
     , style "justify-content" "space-between"
+    , style "border" <|
+        "1px solid "
+            ++ (case state of
+                    StepStateFailed ->
+                        Colors.failure
+
+                    _ ->
```

Comment:
:ghost: Scope creeping, but can we do the same for errored steps? Those are in the same boat: they cause a build failure (well, error) but are hard to notice when auto-collapsed.

I don't think there are any other states we care about. :thinking:

Reply:
Here's a prototype, but let's get @Lindsayauchin in on this sooner rather than later.
![Screen Shot 2019-07-31 at 3 21 45 PM](https://user-images.githubusercontent.com/22056320/62241421-200acd80-b3a7-11e9-9823-6071db7f6855.png)

Reply:
@vito @pivotal-jamie-klassen Makes sense. I like it!

Reply:
@Lindsayauchin @pivotal-jamie-klassen More :ghost: scope creep: what do you think of adding a yellow border for running steps, since they're auto-collapsed too?

some context: https://github.com/concourse/concourse/issues/2608#issuecomment-517296718